### PR TITLE
Fix a bunch of spelling mistakes (3/n)

### PR DESCRIPTION
**Pull request by jsoref**
[master < Epic] PR #1010 
- [ ] Check, and update documentation if necessary
- [ ] Write E2E tests
- [ ] Compare the [benchmarking results](https://bench-graph.memgraph.com/) between the master branch and the Epic branch
- [ ] Provide the full content or a guide for the final git message

[master < Task] PR
- [ ] Check, and update documentation if necessary
- [ ] Provide the full content or a guide for the final git message

To keep docs changelog up to date, one more thing to do:
- [ ] Write a release note here, including added/changed clauses
- [ ] Tag someone from docs team in the comments

### CODE REVIEW

**Comment by gitbuda** [comment]
@jsoref let's please keep at most 3 of these spelling mistake PRs open at any given time because if all of them are created without merging something, we are in the same spot as with one huge PR. In other words, once we merge the existing PRs we can continue to the rest 😄

1/n PR is really close to being merged 😄

**Comment by jsoref** [comment]
Sure.

File: tests/e2e/fine_grained_access/create_delete_filtering_tests.py
Comment (jsoref) [inline tests/e2e/fine_grained_access/create_delete_filtering_tests.py:1]: If the changes to this file are too much, this commit can be split out into its own PR: https://github.com/memgraph/memgraph/commit/fec64817b700084883e863c207fb8b640b2492db

File: src/query/frontend/stripped_lexer_constants.hpp [hunk 1]
```diff
@@ -180,8 +180,8 @@ const trie::Trie kKeywords = {"union",
                               "trigger",
                               "triggers",
                               "update",
-                              "comitted",
-                              "uncomitted",
+                              "committed",
+                              "uncommitted",
```
Comment (gitbuda) [inline src/query/frontend/stripped_lexer_constants.hpp:184]: This change probably won't work

**Comment by gitbuda** [comment]
This PR has a few questionable changes that have to be manually verified, moving this to v2.12 milestone, will try to verify the PR as soon as possible

**Comment by gitbuda** [comment]
Closing this because it's again out of sync + it's not critical.
